### PR TITLE
Reformat code to make it more readable

### PR DESCRIPTION
**Pull request by Dinty1**
Reading through the code, I found it quite hard to follow. I've pretty much just used Intellij's code formatting feature which makes it a bit better, but there are still some things which you may want to address (namely static abuse in places like your managers). If I get a chance, I'll try fix some other things as well.

### CODE REVIEW

**Comment by Dinty1** [comment]
well ok then

**Comment by Dinty1** [comment]
But if you're going to reject people trying to make your code more readable can I suggest that you not go around [calling other peoples' code bad](https://www.spigotmc.org/resources/advanced-cheat-blocker.94084/reviews#review-410245-901405) 🤔

**Comment by LMBishop** [comment]
Don't know why you wasted your time making a PR for someone so toxic lol

**Comment by redaam** [comment]
wow, Such suck

**Comment by SpiritenHasArrived** [comment]
toxic developers will close pr's for no reason sadly

**Comment by Texboobcat** [comment]
hi <3

**Comment by Dinty1** [comment]
> Don't know why you wasted your time making a PR for someone so toxic lol

Was at least worth a try 🤷‍♂️

**Comment by OMGPandaYT** [comment]
Oh did i forget to comment.
Seriously, this is meant for ME to develop and code with no help. So i'm not gonna allow pull requests that modify code.
@Dinty1 your code is good but i'm trying to develop this to learn so i closed it.
Plus spiriten stfu, i'm not toxic and i don't close pr's for no reason.

**Comment by Dinty1** [comment]
1) You could have said that first and then people wouldn't be saying this shit 😛
2) I didn't actually change any code, just made it look nicer

**Comment by OMGPandaYT** [comment]
1. i just forgot to press comment, plus  they still would be saying it because they hate me
2. You changed it to look nicer

**Comment by Dinty1** [comment]
I mean if you want you can ctrl+alt+l it yourself xP

**Comment by OMGPandaYT** [comment]
i sent a message here.. i dont see it.

**Comment by OMGPandaYT** [comment]
Anyways, if it doesn't affect performance it doesn't matter as i can still understand it.
